docs/README.md

```diff
@@ -22,65 +22,103 @@ cd cluster-api-provider-nested
 kind create cluster --name=capn
 ```
 
-### Install `cert-manager`
+### Clone CAPI and build `clusterctl` tool
 
-Cert Manager is a soft dependency for the Cluster API components to enable mutating 
-and validating webhooks to be auto deployed. For more detailed instructions 
-go [Cert Manager Installion](https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests).
+As a cluster API~(CAPI) provider, CAPN need build `clusterctl` tool
 
 ```shell
-kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+git clone git@github.com:kubernetes-sigs/cluster-api.git
+cd cluster-api
+make clusterctl
 ```
 
-### Clone CAPI and Deploy Dev release
+# Create Dev Repo
+mkdir -p ~/.cluster-api/dev-repository/
 
-As a cluster API~(CAPI) provider, CAPN requires core components of CAPI to be setup. 
-We need to deploy the unreleased version of CAPI for `v1alpha4` API support.
+# Update config for CAPI Beta
+cat <<EOF >>~/.cluster-api/dev-repository/config.yaml
+---
+providers:
+- name: "cluster-api"
+  url: "https://github.com/kubernetes-sigs/cluster-api/releases/v0.4.0-beta.0/core-components.yaml"
+  type: "CoreProvider"
+- name: "kubeadm"
+  url: "https://github.com/kubernetes-sigs/cluster-api/releases/v0.4.0-beta.0/bootstrap-components.yaml"
+  type: "BootstrapProvider"
+EOF
```

Comment:
Can you wrap this in 3 backticks to make it a code block?

Reply:
yes, will do